block.go

```diff
@@ -16,8 +16,12 @@ type Disk struct {
 	SizeBytes              uint64
 	PhysicalBlockSizeBytes uint64
 	BusType                string
+	BusPath                string
+	NumaNodeId             int
 	Vendor                 string
+	Model                  string
 	SerialNumber           string
+	WorldWideName          string
```

Comment:
I could support naming this field just "WWN" since that seems to be what most folks would expect/look for?

Reply:
Sounds good to me.